### ISSUE TITLE
Sidebar empty state text does not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52587]
Environment Details:
VSCode Version : 1.24.1
OS Version : Windows 10

Additional Details:
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer

Repro Steps:
1.Launch Visual Studio code desktop application.
2.Navigate to Activity Bar and select "Source Control"(Cntrl+Shift+G) button.
3.Check the color contrast ratio of the content on the page i.e. "There are no active source control providers"

Actual
Color contrast ratio is less than 4.5:1 for the content present on the page.i.e. There are no active source control providers.

Expected:
Color contrast ratio should be more than 4.5:1 for the content present on the page.i.e. There are no active source control providers.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

User Impact:
The End user will not be able to view the content properly.

MAS Reference:
MAS 1.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

Attachment For Reference:
![accessibility a11y_vscode_sourcecontrol_colorcontrast](https://user-images.githubusercontent.com/39204909/41714230-3ad84ab2-756d-11e8-9ead-f63f1099d295.png)

Does this issue occur when all extensions are disabled?: Yes
